src/company_gui.cpp

```diff
@@ -1905,7 +1905,7 @@ struct CompanyInfrastructureWindow : Window
 
 			case WID_CI_ROAD_DESC:
 			case WID_CI_TRAM_DESC: {
-				uint lines = 0;
+				uint lines = 1;
```

Comment:
Could you also add a comment explaining why it needs to start counting at 1?

Reply:
Done.